packages/manager/modules/pci/src/components/project/storages/databases/connectors.constants.js

```diff
@@ -0,0 +1,37 @@
+export const CONNECTOR_STATUS = {
+  CREATING: 'CREATING',
+  RUNNING: 'RUNNING',
+  PAUSED: 'PAUSED',
+};
+export const TASK_STATUS = {
+  RUNNING: 'RUNNING',
+  FAILED: 'FAILED',
+  PAUSED: 'PAUSED',
+};
+export const AVAILABLE_CONNECTOR_TYPES = {
+  SINK: 'sink',
+  SOURCE: 'source',
+};
+
+export const CONNECTOR_FIELDS_FILTER_EDIT = ['name', 'connector.class'];
+
+export const TRANSFORM_CONFIG_GROUP_KEY = 'transformType';
+export const CONNECTOR_CONFIG_GROUP_KEY = 'group';
+export const TRANSFORM_PROPERTY_KEY = 'transform';
+export const EMPTY_GROUP_NAME = 'Other';
+
+export const CONNECTOR_FIELDS_TYPE = {
+  PASSWORD: 'password',
+  INT16: 'int16',
+  INT32: 'int32',
+  INT64: 'int64',
+  BOOLEAN: 'boolean',
+  CLASS: 'class',
+  STRING: 'string',
+  LIST: 'list',
+};
+
+export const CLASS_PROPERTY_AS_STRING = ['header.converter'];
+
+export const TRANSFORM_PROPERTY = 'transforms';
+export const EXTRA_CONFIG_PROPERTY = '*';
```

Comment:
**todo**: Better to add an `export default`

Reply:
Done here: https://github.com/ovh/manager/pull/6576/commits/4f7622e8b1c5a54ff77e71a6974f1bceae060d45